addons/web_editor/static/src/js/wysiwyg/widgets/link_tools.js

```diff
@@ -44,14 +44,24 @@ const LinkTools = Link.extend({
     /**
      * @override
      */
-    start: function () {
+    start: async function () {
         const titleEls = this.el.querySelectorAll('we-title');
         for (const titleEl of titleEls) {
             // See _buildTitleElement for explanation
             titleEl.textContent = titleEl.textContent.replace(/⌙/g, '└');
         }
         this._addHintClasses();
-        return this._super(...arguments);
+        const ret = await this._super(...arguments);
+        const link = this.$link[0];
+        const customStyleProps = ['color', 'background-color', 'background-image', 'border-width', 'border-style', 'border-color'];
+        if (customStyleProps.some(s => link.style[s])) {
+            // Force custom style if style exists on the link.
+            const options = this.el.querySelectorAll('[name="link_style_color"] we-button');
+            const customOption = [...options].find(opt => opt.dataset.value === 'custom');
```

Comment:
You seem to search all buttons to then find only one inside. Is not this equal to:
```suggestion
            const customOption = this.el.querySelector('[name="link_style_color"] we-button[data-value="custom"]');
```
?

Reply:
You're absolutely right of course :facepalm: Thanks.